_rest_routes.inc.php

```diff
@@ -328,6 +329,10 @@
 
 );
 
+$restApiExtendEvent = new RestApiExtendEvent(RestConfig::$ROUTE_MAP);
+$restApiExtendEvent = $GLOBALS["kernel"]->getEventDispatcher()->dispatch(RestApiExtendEvent::EVENT_HANDLE, $restApiExtendEvent, 10);
```

Comment:
Do you plan on only extending the REST api, or will you be extending the FHIR api also?  @sjpadgett do you have any thoughts on this?

You could leverage the same event for both the REST and FIHR pieces, but you would need to pass in some context to say which one is being extended...

If you don't care about FIHR we can treat it as a separate event.

Reply:
Does it matter? Which api used is determined by the route.

Reply:
@sjpadgett you're right that the api used is determined by the route, but this pull request is attempting to modify just the routes in $ROUTE_MAP and not the ones in $FHIR_ROUTE_MAP.

Reply:
duh Jerry! Right, Yes we'd want FHIR inclusion. I am wondering now if I should have just designed for one route array. I was thinking it more efficient memory wise to separate them.

Reply:
In the next commit, I will try to implement this feature.